scripts/pylib/twister/twisterlib/testplan.py

```diff
@@ -589,6 +600,8 @@ def load_from_file(self, file, filter_platform=[]):
                     instance.status = status
                     instance.reason = reason
 
+                self.handle_quarantined_tests(instance, platform)
```

Comment:
with the intention to 
> .. sometime we need use quarantine at runtime when --test-only

wouldn't it better to filter outside of `load_from_file()`, at `load()` ? just near this line -
https://github.com/zephyrproject-rtos/zephyr/blob/33c253c813d8cc56209ce0b572e127cd6eaa941b/scripts/pylib/twister/twisterlib/testplan.py#L236

Reply:
@golowanow , my initial idea is to put the related code close together. As in load_from_file, there is already has filterred porcessing there, so I put the similiar code there.  Also if we move it outside, we need loop on the instance_list again.